src/components/sbb-pearl-chain-vertical-item/sbb-pearl-chain-vertical-item.scss

```diff
@@ -198,6 +198,17 @@
   vertical-align: top;
 }
 
+.sbb-pearl-chain-vertical-item__column--right {
+  width: var(--sbb-pearl-chain-vertical-right-item-width, 100%);
```

Comment:
It would be more consistent if you just assign 100% to --sbb-pearl-chain-vertical-right-item-width variable on :host().